debug/test/browser/debug-suspense.test.js

```diff
@@ -106,8 +106,8 @@ describe('debug with suspense', () => {
 
 				return loader.then(() => {
 					rerender();
-					expect(console.warn).to.be.calledTwice;
-					expect(warnings[1].includes('MyLazyLoaded')).to.equal(true);
+					expect(console.warn).to.be.calledThrice;
+					expect(warnings[2].includes('MyLazyLoaded')).to.equal(true);
```

Comment:
Since lazy is re-rendered when mounting the fallback, these checks get triggered an additional time.

Reply:
This one seems curious to me: we invoke the component that threw in order to un-render it?